internal/pkg/manifest/env_test.go

```diff
@@ -801,3 +819,65 @@ func TestEnvironmentObservability_IsEmpty(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvironmentCDNConfig_IsEmpty(t *testing.T) {
+	testCases := map[string]struct {
+		in     environmentCDNConfig
+		wanted bool
+	}{
+		"empty": {
+			in:     environmentCDNConfig{},
+			wanted: true,
+		},
+		"not empty": {
+			in: environmentCDNConfig{
+				Enabled: aws.Bool(false),
+			},
+			wanted: false,
+		},
+	}
+
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			got := tc.in.IsEmpty()
+			require.Equal(t, tc.wanted, got)
+		})
+	}
+}
+
+func TestEnvironmentCDNConfig_CDNEnabled(t *testing.T) {
+	testCases := map[string]struct {
+		in     environmentCDNConfig
+		wanted bool
+	}{
+		"enabled via bool": {
+			in: environmentCDNConfig{
+				Enabled: aws.Bool(true),
+			},
+			wanted: true,
+		},
+		// "enabled via struct": {
+		// 	in: environmentCDNConfig{
+		// 		CDNConfig: AdvancedCDNConfig{},
+		// 	},
+		// 	wanted: true,
+		// },
```

Comment:
🤔  can we get rid of unused code?

Reply:
There's similarly commented code in `transform_test.go` that I can get rid of too